app/controllers/user_applications_controller.rb

```diff
@@ -61,7 +60,7 @@ def create
       redirect_to success_user_applications_path
       return
```

Comment:
@SeanLF might not need the "return" here as redirect_to should halt execution. Might be wrong.